Prodigy entity linking recipe

michel-ds opened the pull request:
Adds the Prodigy entity linking recipe and configuration that allows clicking on an entity link without automatically selecting that option.

michel-ds committed:
Initial commit of Prodigy entity linking recipe

michel-ds committed:
Fix typo

Luis-R-Flores commented on prodigy/Readme.md:
````diff
@@ -0,0 +1,10 @@
+Please ensure that you run this code in the same directory in order to allow Prodigy to pick up the local `prodigy.json` file.
+
+```bash
+ python -m prodigy entity_linker.manual <dataset> <source_file> <nlp_model> <kb_file> <additional_info_entities_file -F el_recipe.py>
````
Correct to: 

#
python -m prodigy entity_linker.manual <dataset> <source_file> <nlp_model> <kb_file> 
<additional_info_entities_file> -F el_recipe.py 
#

??

michel-ds committed:
Speed-up: Load less of nlp model

michel-ds committed:
Open Guardian link in new tab

michel-ds committed:
Allow installed Spacy model to be used or load model from path

michel-ds committed:
Better labels for annotation options

michel-ds committed:
Improved UI display, including formatting and candidate paragraph truncating

This includes the changes made in branch `lf_jai_data_cleaning_and_prodigy_recipe` (commit 4ccab0ad697041aa9dad699cc593168da3dfd8cb)

Luis-R-Flores committed:
Reorder no candidate options. Revert to default primary UI font.

michel-ds committed:
Merge pull request #2 from guardian/lf/prodigy-recipe

Reorder no candidate options. Revert to default primary UI font.

michel-ds committed:
Add instructions to UI.

Luis-R-Flores committed:
Improve kb description UI output length

michel-ds committed:
Merge pull request #3 from guardian/lf/prodigy-recipe

Improve kb description UI output length

michel-ds committed:
Move to JSONL input and faster streaming via generators

This requires a changed data input format as JSON-lines with only two keys:
text and url

Luis-R-Flores reviewed (approved):
Bril!
